app/views/user_added_to_project_mailer/added_to_project.text.erb

```diff
@@ -0,0 +1,13 @@
+You have been added to the Zooniverse project <%= @project.display_name %> in the following roles: <%= @roles.join(" and ") %>
+
+Visit <a href="https://zooniverse.org/projects/<%= @project.slug %>"><%= @project.display_name %></a> to get started.
```

Comment:
Look at using the [`Panoptes.frontend_configuration.frontend_url`](https://github.com/zooniverse/Panoptes/blob/master/config/initializers/redirect_urls.rb#L11) here if you can as it'll work on staging / prod equally well.